.env.template

```diff
@@ -0,0 +1,27 @@
+# OpenShift
+THOTH_CEPH_BUCKET=
+THOTH_CEPH_BUCKET_PREFIX=
+THOTH_S3_ENDPOINT_URL=
+THOTH_DEPLOYMENT_NAME=
+KUBERNETES_VERIFY_TLS=
+THOTH_AMUN_INSPECTION_NAMESPACE=
+THOTH_AMUN_INFRA_NAMESPACE=
+THOTH_FRONTEND_NAMESPACE=
+THOTH_MIDDLETIER_NAMESPACE=
+THOTH_BACKEND_NAMESPACE=
+THOTH_INFRA_NAMESPACE=
+KUBERNETES_API_URL=
+OPENSHIFT_API_URL=
+THOTH_USE_ARGO=
+
+# Logging
+THOTH_LOG_SOLVER=
+THOTH_LOG_SOLVER__PYTHON=
```

Comment:
These two are not used within common code base - they are specific configuration when configuring logging.